tensorboard/webapp/runs/views/runs_table/runs_data_table.ng.html

```diff
@@ -19,8 +19,11 @@
   [headers]="headers"
   [sortingInfo]="sortingInfo"
   [columnCustomizationEnabled]="true"
+  [selectableColumns]="selectableColumns"
   (sortDataBy)="sortDataBy.emit($event)"
   (orderColumns)="orderColumns.emit($event)"
+  (addColumn)="addColumn.emit($event)"
+  (removeColumn)="removeColumn.emit($event)"
```

Comment:
Have you considered creating a single "editColumns" event which could encompass order, add, and remove. It could simply pass the new list of enabled headers. 

This may be a lot harder to do at the moment. It would be easier when we remove the enabled boolean.

If we do this I think it should be a refactor later.